tests/test_auth_service.py

```
@@ -0,0 +1,116 @@
+    )
+    assert auth.get_user() == {"username": "admin"}
+
+
+# ---------- fail closed：损坏的凭据文件绝不静默重置账户 ----------
+
+
+def test_corrupt_json_fails_closed(tmp_path):
+    store = _store(tmp_path)
+    store.write_text("{not valid json", encoding="utf-8")
+    with pytest.raises(AuthStoreError):
+        AuthService(store, default_username="admin", default_password="pw")
```

Ruff (0.15.17)

[error] Possible hardcoded password assigned to argument: "default_password"

(S106)